rosa_planning/rosa-sts-aws-prereqs.adoc

```diff
@@ -50,12 +50,28 @@ include::modules/rosa-sts-aws-requirements-security-req.adoc[leveloffset=+2]
 include::modules/rosa-sts-aws-requirements-ocm.adoc[leveloffset=+2]
 include::modules/rosa-sts-aws-requirements-association-concept.adoc[leveloffset=+3]
 include::modules/rosa-sts-aws-requirements-creating-association.adoc[leveloffset=+3]
+
+[discrete]
+[role="_additional-resources"]
+[id="additional-resources_creating-association_{context}"]
+== Additional resources
+* See xref:../rosa_architecture/rosa-sts-about-iam-resources.adoc#rosa-sts-account-wide-roles-and-policies_rosa-sts-about-iam-resources[Account-wide IAM role and policy reference] for a list of IAM roles needed for cluster creation.
+
 include::modules/rosa-sts-aws-requirements-creating-multi-association.adoc[leveloffset=+3]
 
 
 include::modules/rosa-requirements-deploying-in-opt-in-regions.adoc[leveloffset=+1]
 include::modules/rosa-setting-the-aws-security-token-version.adoc[leveloffset=+2]
-include::modules/rosa-sts-aws-iam.adoc[leveloffset=+1]
+
+[id="rosa-sts-policy-iam_{context}"]
+== Red Hat managed IAM references for AWS
+
+With the STS deployment model, Red Hat is no longer responsible for creating and managing Amazon Web Services (AWS) IAM policies, IAM users, or IAM roles. For information on creating these roles and policies, see the following sections on IAM roles.
+
+* To use the `ocm` CLI, you must have an `ocm-role` and `user-role` resource. See xref:../rosa_planning/rosa-sts-ocm-role.adoc#rosa-sts-ocm-role[OpenShift Cluster Manager IAM role resources].
```

Comment:
Resource or role?

I think the user needs to have both these roles to user the CLI.